tests/models/rl/unit/test_ppo.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+from unittest import TestCase
+
+import numpy as np
+import torch
+
+from pl_bolts.models.rl.ppo_model import PPO
+
+
+class TestPPOCategorical(TestCase):
```

Comment:
pls do not deprecated unitest, rather a simple test functions

Reply:
sure but i don't get what you mean. can you elaborate? 

followed the format of [test_reinforce](https://github.com/PyTorchLightning/lightning-bolts/blob/master/tests/models/rl/unit/test_reinforce.py) for this

Reply:
ah okay, got it. just pushed commits for this. lmk if that works